packages/SwingSet/src/devices/plugin.js

```diff
@@ -24,7 +23,7 @@ export function buildPlugin(pluginDir, pluginRequire, queueThunkForKernel) {
   return {
     srcPath,
     endowments: {
-      require: pluginRequire,
+      import: importPlugin,
```

Comment:
I assume that any "plugins" defined elsewhere in `agoric-sdk` have been updated to stop using `require` in favor of `import`?